Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6
+FROM python:3.7
```

Comment:
Not an issue, just wondering why this is necessary

Reply:
No issue. Upgraded to match Caliban.